src/Server/Query.elm

```diff
@@ -7,8 +7,10 @@ module Server.Query exposing
 
 import Data.Country as Country exposing (Country)
 import Data.Env as Env
-import Data.Food.Db as FoodDb
-import Data.Food.ExplorerRecipe as Recipe
+import Data.Food.Builder.Db as BuilderDb
+import Data.Food.Builder.Query as BuilderQuery
+import Data.Food.Ingredient as Ingredient
```

Comment:
You should `import Data.Food.Ingredient as Ingredient exposing (Ingredient)` to avoid repeating `Ingredient.Ingredient` in the code